libraries/helpers.rb

```diff
@@ -48,7 +48,7 @@ def command(config_file, config_dir)
       if windows?
         %{agent -config-file="""#{config_file}""" -config-dir="""#{config_dir}"""}
       else
-        "/usr/bin/env consul agent -config-file=#{config_file} -config-dir=#{config_dir}"
```

Comment:
I've done this because `poise-service` splits the "command" on two parts by the first space:
https://github.com/poise/poise-service/blob/3355a9d1ae72761c3da36e72ed8b9912b8fb5be9/lib/poise_service/service_providers/sysvinit.rb#L53-L69

Thus, in the template we will be able to use variables `@daemon` and `@daemon_options`. The value of `@daemon` is something like `/usr/local/bin/consul`, e.q. it is already expanded to absolute path.